e2e_tests/tests/per_provider/normal_tests/import_key.rs

```diff
@@ -238,6 +320,7 @@ fn check_format_import3() -> Result<()> {
     Ok(())
 }
 
+#[cfg(not(feature = "cryptoauthlib-provider"))]
 #[test]
 fn failed_imported_key_should_be_removed() -> Result<()> {
```

Comment:
Instead of skipping the test completely, any chance you could make `public_key` and `attributes` conditional and have it try to import an ECC key for CALib? Or even just modifying the last import call to succeed in the CAL provider

Reply:
Done